### DISCUSSION TITLE
SKImageFilter.CreatePaint is missing in 3.0.0-alpha.1.7

### DISCUSSION THREAD

**Discussion by wieslawsoltes** [mono/SkiaSharp#2569] [Q&A] [answered]
I am testing 3.0.0-alpha.1.7 version in my Svg.Skia library and found SKImageFilter.CreatePaint api is missing. Is there alternative api I can use.

**Comment by mattleibow** [comment, marked as answer]
According to the changelog from Google 

* Removed previously deprecated SkImageFilters::Paint factory. Use SkImageFilters::Shader instead.

I think the paint was really deceiving as only the Shader and Color properties were ever used. Think:

```
var shader = paint.Shader ?? SKShader.CreateColor(paint.ColorF);
var filter = SKImageFilter.CreateShader(shader, paint.IsDither);
```

https://github.com/google/skia/commit/7d0f8531580fc07a70e62c33549f64c6db691677
https://github.com/google/skia/commit/89f8ba6f749d89fe201c812cc3a67fbf4db6572b

**Comment by wieslawsoltes** [comment]
> According to the changelog from Google
> 
> * Removed previously deprecated SkImageFilters::Paint factory. Use SkImageFilters::Shader instead.
> 
> I think the paint was really deceiving as only the Shader and Color properties were ever used. Think:
> 
> ```
> var shader = paint.Shader ?? SKShader.CreateColor(paint.ColorF);
> var filter = SKImageFilter.CreateShader(shader, paint.IsDither);
> ```
> 
> [google/skia@7d0f853](https://github.com/google/skia/commit/7d0f8531580fc07a70e62c33549f64c6db691677) [google/skia@89f8ba6](https://github.com/google/skia/commit/89f8ba6f749d89fe201c812cc3a67fbf4db6572b)



> According to the changelog from Google
> 
> * Removed previously deprecated SkImageFilters::Paint factory. Use SkImageFilters::Shader instead.
> 
> I think the paint was really deceiving as only the Shader and Color properties were ever used. Think:
> 
> ```
> var shader = paint.Shader ?? SKShader.CreateColor(paint.ColorF);
> var filter = SKImageFilter.CreateShader(shader, paint.IsDither);
> ```
> 
> [google/skia@7d0f853](https://github.com/google/skia/commit/7d0f8531580fc07a70e62c33549f64c6db691677) [google/skia@89f8ba6](https://github.com/google/skia/commit/89f8ba6f749d89fe201c812cc3a67fbf4db6572b)

Ah yes that might work in my case.
